src/python/artm/_js/ldavis.js

```diff
@@ -0,0 +1,1361 @@
+LDAvis = function(to_select, data) {
```

Comment:
What is the license for ldavis? Please check that it follows a permissive license (MIT, BSD-3-clause, or similar). Consider adding ldavis license at the bottom of BigARTM LICENSE file (in the root of the project)